contracts/BSCValidatorSet.sol

```diff
@@ -114,8 +126,13 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
         validatorExtraSet.push(validatorExtra);
       }
     }
+    _;
+  }
 
+  modifier oncePerBlock() {
+    require(block.number > previousHeight, "can not slash twice in one block");
```

Comment:
`"can not slash twice in one block"` description is wrong.

Reply:
Fixed